### ISSUE TITLE
About finetune DTU scene

### ISSUE THREAD

**Issue by starhiking** [apchenstu/mvsnerf#15]
When fine-tuning the DTU scene,  I find only 114 is provided the download link,  and how to download the other scene dataset

**Comment by apchenstu** [comment]
please refer to the "Data download" instruction

**Comment by starhiking** [comment]
Hi, I have downloaded DTU and DTU example for finetune.

Fine-tuning 114 scene from DTU example works well, while finetuning from DTU occurs problems as follows:

First, I need change the depth file path in dtu_ft.py. (dirnames are Depths/scan114_train in DTU, and Depths/scan114 in DTU example)

Then, I meets the problem the resolution of depth image is 128x160, which cannot work at the read_depth function.
    
    def read_depth(self, filename):
            depth_h = np.array(read_pfm(filename)[0], dtype=np.float32)  # (800, 800)
            depth_h = cv2.resize(depth_h, None, fx=0.5, fy=0.5,
                                 interpolation=cv2.INTER_NEAREST)  # (600, 800)
            depth_h = depth_h[44:556, 80:720]  # (512, 640)
            depth_h = cv2.resize(depth_h, None, fx=self.downsample, fy=self.downsample,
                                 interpolation=cv2.INTER_NEAREST)  # !!!!!!!!!!!!!!!!!!!!!!!!!
    
            return depth_h


So what should I do for this case?

**Comment by starhiking** [comment]
Should replace the Depths (in DTU) with  raw data?

**Comment by apchenstu** [comment]
Yeah, the resolution of our depth map is the same as the raw data

**Comment by dwangxxx** [comment]
> Should replace the Depths (in DTU) with raw data?

Hello，请问一下你finetuning的结果是正常的吗？我这边finetuning的结果比general的还要差，不知道什么原因，可以交流一下吗？

**Comment by JJJano** [comment]
> > Should replace the Depths (in DTU) with raw data?
> 
> Hello，请问一下你finetuning的结果是正常的吗？我这边finetuning的结果比general的还要差，不知道什么原因，可以交流一下吗？

hi，请问你复现出finetuning结果了吗，我也得到了比general更差的结果

**Comment by hengfei-wang** [comment]
> 

我也是啊，怎么finetuning的结果比general差这么多。我用的是LLFF数据

**Comment by Shichao123123** [comment]
> 请问您解决了这个问题吗？我的finetuning效果也比论文里面呈现的要差很多
